pkg/ui/workspaces/db-console/src/views/cluster/containers/timeScaleDropdownWithSearchParams/index.tsx

```diff
@@ -21,15 +21,65 @@ import { PayloadAction } from "src/interfaces/action";
 import { AdminUIState } from "src/redux/state";
 import * as timewindow from "src/redux/timeScale";
 
+// toQueryParamTimescale converts a preset name to a query param
+// format, replacing spaces with dashes.
+function toQueryParamTimescale(str: string): string {
+  return str.toLowerCase().replace(/\s+/g, "-");
+}
+
+export const FindPreset = (s: string): string | undefined => {
```

Comment:
nit: I think in JS the convention is not to capitalize exported functions unless they are react components. I know that's not followed in a bunch of legacy code atm but I feel like we should try to stick to this moving forward, lmk if you disagree.

Sidenote, I can't find places where this is used outside the file unless I missed something, so we might not need to export anything.

Reply:
Done.